eks_infra/install_cluster_components.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+#/ Usage: 
+#/ export AWS_REGION=<AWS-Region>
+#/ ./install_cluster_components.sh eks_cluster_name cluster_autoscalar_image_version
+
+set -e
+
+# Function to install cluster autoscalar
+function install_cluster_autoscalar(){
+  kubectl apply -f cluster-autoscalar-autodiscover.yaml
+  kubectl -n kube-system annotate deployment.apps/cluster-autoscaler cluster-autoscaler.kubernetes.io/safe-to-evict="false"
+  sed -e 's/<CLUSTER_NAME>/'"${1}"'/g;s/<VERSION>/'"${2}"'/g' cluster-autoscalar-autodiscover.yaml > /tmp/cluster-autoscalar-autodiscover-${1}.yaml &&
+  kubectl replace -f /tmp/cluster-autoscalar-autodiscover-${1}.yaml
+
+}
+
+# Check for input arguments
+if [ $# -ne 2 ]; then
+    echo "${0}: usage: ./install_cluster_components.sh eks_cluster_name cluster_autoscalar_image_version"
+    exit 1
+fi
+
+# Check for environment variables
+if [ -z "$AWS_REGION" ]; then
+  echo "AWS region not configured"
+  exit 1
+fi
+
+CLUSTER_NAME=${1}
+CLUSTER_AUTOSCALAR_IMAGE_VERSION=${2}
+REGION=${AWS_REGION}
```

Comment:
As mentioned in previous comments, AWS_REGION is a necessary input, so you can use this variable directly in the rest of the script.